pulp_2_tests/tests/puppet/api_v2/test_sync_publish.py

```diff
@@ -88,7 +91,7 @@ def test_importer_config(self):
                 self.assertEqual(body['importer_' + key], importers[0][key])
 
```

Comment:
This class will need a `TearDownClass` implementing the cleanup of `cls.resources.`